### DISCUSSION TITLE
关于 InnerSource 的资料收集

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#114] [日常讨论]
组织：
- InnerSource Commons：https://innersourcecommons.org/
- GitHub InnerSource：https://resources.github.com/innersource/
- https://en.wikipedia.org/wiki/Inner_source
- 

视频：

- What is Inner Source?：https://dev.to/n3wt0n/inner-source-model-explained-321e
- InnerSource & DevOps：https://www.linkedin.com/pulse/innersource-devops-event-takeaways-cristina-coffey/

文章：
- What is InnerSource?：https://about.gitlab.com/topics/version-control/what-is-innersource/
- An introduction to innersource：https://resources.github.com/innersource/fundamentals/
- Building an InnerSource ecosystem using AWS DevOps tools：https://aws.amazon.com/cn/blogs/devops/building-an-innersource-ecosystem-using-aws-devops-tools/

教程：
- Manage an InnerSource program by using GitHub：https://learn.microsoft.com/en-us/training/modules/manage-innersource-program-github/

**Comment by wangleo61** [comment]
WoW，Thanks Wang Sir’s sharing.

Looks good stuff.

BR

Leo

On Feb 4, 2023, at 09:07, Will Wang ***@***.***> wrote:



组织：

  *   InnerSource Commons：https://innersourcecommons.org/
  *   GitHub InnerSource：https://resources.github.com/innersource/
  *   https://en.wikipedia.org/wiki/Inner_source
  *

视频：

  *   What is Inner Source?：https://dev.to/n3wt0n/inner-source-model-explained-321e
  *   InnerSource & DevOps：https://www.linkedin.com/pulse/innersource-devops-event-takeaways-cristina-coffey/

文章：

  *   What is InnerSource?：https://about.gitlab.com/topics/version-control/what-is-innersource/
  *   An introduction to innersource：https://resources.github.com/innersource/fundamentals/
  *   Building an InnerSource ecosystem using AWS DevOps tools：https://aws.amazon.com/cn/blogs/devops/building-an-innersource-ecosystem-using-aws-devops-tools/

教程：

  *   Manage an InnerSource program by using GitHub：https://learn.microsoft.com/en-us/training/modules/manage-innersource-program-github/

**Reply by will-ww** [reply]
> WoW，Thanks Wang Sir’s sharing. Looks good stuff. 

一个考虑是，今年准备在 InnerSource 这块也做点事情，包括知识的生产、宣传与工具化三个方面。有兴趣的可以一起来搞搞~ :-)

**Comment by wangleo61** [comment]
好呀，王老师一起哈。

On Feb 5, 2023, at 09:42, Will Wang ***@***.***> wrote:



WoW，Thanks Wang Sir’s sharing. Looks good stuff.

一个考虑是，今年准备在 InnerSource 这块也做点视频，包括知识生产、宣传与工具化三个方面。有兴趣的可以一起来搞搞~ :-)

**Comment by coolbeevip** [comment]
还有这个 https://patterns.innersourcecommons.org/v/zh/
